templates/tco-calculator/index.html

```diff
@@ -10,8 +10,12 @@
 <section class="p-strip--suru is-dark">
   <div class="row">
     <div class="col-8">
-      <h1>Private cloud TCO calculator</h1>
-      <p>One of the biggest promises behind the private cloud infrastructure is better economics compared to proprietary hardware and public clouds. Use the TCO (total cost of ownership) calculator below to estimate the cost of your private cloud infrastructure with Canonical.</p>
+      <h1>
+        Private cloud TCO calculator
+      </h1>
+      <p>
+        One of the biggest promises behind the private cloud infrastructure is better economics compared to proprietary hardware and public clouds. Use the TCO (total cost of ownership) calculator below to estimate the cost of your private cloud infrastructure with Canonical or <a class="p-link--inverted" href="/openstack/compare">learn more about differences between various OpenStack distributions</a>.
```

Comment:
You can't see that this is a link, as it is the same colour as the rest of the text.

Reply:
That's a Vanilla thing? sorry

Reply:
Sweet!